### PR TITLE
Password reuse

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: Security/Fundamentals.mediawiki [hunk 1]
```diff
@@ -22,10 +22,20 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
 |-
+|<div id="password-reuse">[[#password-reuse|§]] Password reuse</div>
+|Password reuse is the practice of a single user using the same password across multiple different accounts/sites. This is contrasted with creating a different, distinct password for every account/site. Users often employ hybrid forms of password reuse like
+* Using the same password for a class of accounts/sites, for example, using one single password for multiple high value financial accounts, but a different single password for multiple low value forums and wikis.
+* Using a consistent reproducible method of password generation for each site, for example, every account/site has a password which begins with the same characters and ends with name of the site ("rosebud0facebook", "rosebud0linkedin")
+Password reuse is discouraged because:
+* When a site is compromised by an attacker, the attacker can easily take the user's password with has been reused on other sites and gain access to those other sites. For example if a user uses the same password on a car forum website as they use on Facebook, when that car website gets compromised, the attackers can then takeover the user's Facebook account.
```
Comment (Phrozyn) [inline Security/Fundamentals.mediawiki]: change 'with' to 'that'?
When a site is compromised by an attacker, the attacker can easily take the user's password _with_ has been reused

**Comment by gdestuynder** [comment]
> the attacker can easily take the user's password with has been reused on 
seems to be a typo / missing part in the sentence

**Comment by gene1wood** [comment]
@gdestuynder Yup, good catch, fixed in f1a649b

**Comment by gdestuynder** [comment]
L33: I would say "when a site's user password" or something similar for clarity, that we assume the attacker has the password, since else there may be confusions and discussions such as: "but its hashed" etc.

**Comment by gene1wood** [comment]
@gdestuynder Can you give me a little more info, what's the line look like with your suggested edit? Also, is the section that talks about hashing enough to address this?

File: Security/Fundamentals.mediawiki [hunk 2]
```diff
@@ -22,10 +22,20 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
+* Using the same password for a class of accounts/sites, for example, using one single password for multiple high value financial accounts, but a different single password for multiple low value forums and wikis.
+* Using a consistent reproducible method of password generation for each site, for example, every account/site has a password which begins with the same characters and ends with name of the site ("rosebud0facebook", "rosebud0linkedin")
+Password reuse is discouraged because:
+* When a site is compromised by an attacker, the attacker can easily take the user's password that has been reused on other sites and gain access to those other sites. For example if a user uses the same password on a car forum website as they use on Facebook, when that car website gets compromised, the attackers can then takeover the user's Facebook account.
+* For every site that a user has reused a password at, any unethical administrators of any of those sites has the ability to access that user's accounts on other sites where the user has reused the password.
```
Comment (Phrozyn) [inline Security/Fundamentals.mediawiki]: Maybe consider revising to:

Unethical administrators of any sites where a password is reused may/can gain access to accounts using the reused password.

Comment (Phrozyn) [inline Security/Fundamentals.mediawiki]: Maybe consider revising to:

Unethical administrators of any sites where a password is reused may/can gain access to accounts using the reused password.

Comment (gene1wood) [inline Security/Fundamentals.mediawiki]: That's much less awkward than what I wrote. I'll use your suggestion.

**Pull request merged**